src/Symfony/Installer/NewCommand.php

```diff
@@ -133,7 +139,22 @@ private function checkSymfonyVersionIsInstallable()
     {
         // 'latest' is a special version name that refers to the latest stable version
         // available at the moment of installing Symfony
-        if ('latest' === $this->version) {
+        if (self::VERSION_LATEST === $this->version) {
+            return $this;
+        }
+
+        // Get the Symfony versions list (http://symfony.com/versions.json)
+        $versionsList = $this->getVersionsList();
```

Comment:
@94noni what you should do is move this call inside the ``if (self::VERSION_LTS === $this->version) {``  conditional (and add it again in the place where you removed it). The call won't be done twice anyway because the JSON gives you the exact version for the LTS, not the branch